### PR TITLE
charts-review

### PR DESCRIPTION
**Pull request by kghamilton89**
This PR fixes broken example sections and also contains a general review because I came across some grammar errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 ## Overview
 
-Charts is a collection of time-series [widgets](./widgets/README.md) presented in grid layout.
+The **Charts Library** is a collection of time-series [widgets](./widgets/README.md) presented in a grid layout.
```
Comment (rodionos) [inline README.md]: Move this into separate PR, so that format fixes and major content changes can be reviewed separately.

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The Charts library implements a simple yet powerful syntax, closely integrated w
 
 ## Widgets
 
-All widgets support [Shared Widget](./widgets/shared/README.md) syntax.
+All widgets support either [Shared Widget](./widgets/shared/README.md) or [Shared Table Widget](./widgets/shared-table/README.md) syntax.
```
Comment (rodionos) [inline README.md]: Move to separate PR - the new sentence is incorrect.

File: syntax/control-structures.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ If the elements fit on one line, `endlist` is **not** required.
 list servers = awsswgvml001, nurswgvml003
 ```
 
-The `list` command is similar to [`var`](#var--endvar), without having to quote and enclose elements in square brackets.
+The `list` command is similar to [`var`](#var--endvar) without the need to quote and enclose elements in square brackets.
```
Comment (rodionos) [inline syntax/control-structures.md]: list does not require to enclose elements in quotes. This is not clear from the sentence.

File: syntax/control-structures.md [hunk 4]
```diff
@@ -191,7 +191,7 @@ for server in servers
 endfor
 ```
 
-Use `elementname_index` without the control symbol `@` and use curly brackets to access current iterator index starting with `0`.
+Use `elementname_index` without the control symbol `@` and curly brackets to access current iterator index starting with `0`.
```
Comment (rodionos) [inline syntax/control-structures.md]: > Use ... without ... and 
Unclear

File: syntax/control-structures.md [hunk 5]
```diff
@@ -207,7 +207,7 @@ for server in servers
 endfor
 ```
 
-Similar layout with automated row breaks can be accomplished by specifying the `widgets-per-row` setting under `[group]` level.
+Create a similar layout with automated row breaks by specifying the `widgets-per-row` setting at the `[group]` level.
```
Comment (rodionos) [inline syntax/control-structures.md]: > automated row breaks
Unclear

File: syntax/format-settings.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ Value | Syntax | Description
 `binary` | `format = binary('kilo', 3) + 'B'` | Formatting values with size units. Used for bytes.<br>For example, `1024`.
 `decimal`| `format = numeric(1)`<br>`format = numeric`| Number formatting: `numeric` is an `alias` for `decimal`.<br>Specify the optional round value parameter in round brackets to define how many spaces the decimal is rounded.<br>Example: `format = numeric(1)` rounds to the first decimal. Thus, `4.29` becomes `4.3`.
 `fixed`| `format = fixed(3)` | Format numbers with the specified number of digits.<br>Example: `format = fixed(1)` rounds `1225` to `1000`.
-`currency` | `format = '$' + currency('million')`| Currency formatting. Any symbol is supported.
+`currency` | `format = '$' + currency('million')`| Currency formatting.<br>Any symbol is supported.
```
Comment (rodionos) [inline syntax/format-settings.md]: > Any symbol is supported.
Perhaps any **currency** symbol. what happens to the number itself? Describe.

File: syntax/format-settings.md [hunk 7]
```diff
@@ -153,7 +153,7 @@ interval-format = %H:%M
 
 ## Day Format
 
-Control the date format of the `x` axis for year, month, week, and day.
+Display the date format of the `x` axis as a year, month, week, or day.
```
Comment (rodionos) [inline syntax/format-settings.md]: > Display the date format
Should it be 'Modify the date format...'?

File: syntax/selecting-series.md [hunk 8]
```diff
@@ -290,4 +290,4 @@ display = value == top(1) || value == bottom(1)
 
 [![](./images/button.png)](https://apps.axibase.com/chartlab/cdfb34c5/13/)
 
-In addition, the `limit = int` setting reduces the number of samples displayed for each series. It makes queries execute faster when loading data for high-frequency series from the server, in particular during design and validation stages.
\ No newline at end of file
+In addition, the `limit` setting reduces the number of samples displayed for each series. This aids in quicker query execution when loading data for high-frequency series from the server, in particular during design and validation stages.
```
Comment (rodionos) [inline syntax/selecting-series.md]: Avoid 'this' and pronouns.

File: syntax/thresholds.md [hunk 9]
```diff
@@ -2,11 +2,11 @@
 
 Use client-side threshold aggregators to compute the amount of time or number of occurrences a metric violates a user-defined threshold.
 
-Specify lower or upper thresholds or combine the two to trigger a violation when a series value is exceeds the specified range.
+Specify upper and lower thresholds or a combination of the two to trigger violations when a series value exceeds the specified range.
```
Comment (rodionos) [inline syntax/thresholds.md]: > to trigger violations
? Do you mean to emphasize areas of the chart or specific samples to illustrate violations?

File: syntax/udf.md [hunk 10]
```diff
@@ -136,11 +136,11 @@ Server restart is **not** required to access new or updated function files.
 
 ## Writing Functions
 
-Function declaration must start with `exports.` followed by a valid function name. Function names are **case-sensitive**.
+Function declaration must start with `exports.` followed by a valid function name. Function names are **case sensitive**.
```
Comment (rodionos) [inline syntax/udf.md]: Use hyphen for case-sensitive

File: widgets/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Widgets
 
-All widgets support [Shared Widget](./shared/README.md) syntax.
+All widgets support either [Shared Widget](./shared/README.md) or [Shared Table Widget](./shared-table/README.md) syntax.
```
Comment (rodionos) [inline widgets/README.md]: Incorrect

File: widgets/text-widget/README.md [hunk 12]
```diff
@@ -23,7 +23,7 @@ Inherited settings from [Shared Widget](../shared/README.md)): `entity`, `metric
...
+|Icon|`icon = public/img/svg/linear/users.svg`<br>`icon = alerts_color_01`, `icon = alerts-color-01`|Add an `svg` icon.Built-in icons can be referenced by name.<br>View built-in [Icon Reference Sheet](../pie-chart/resources/atsd-embedded-icons.pdf).<br>Substitute underscores (`_`) in built-in icon names with dashes(`-`):<br>`icon = alerts_color_07`<br>`icon = alerts-color-07`. Place custom icons in the `/opt/atsd/atsd/conf/portal/svg` directory and refer to them with the `/portal/resource/svg/image` `.svg` path.<br>Custom icons in Axibase Server must be store in `./AxibaseServer/config/workspace/svg` and referenced with `/workspace/svg/image` `.svg` path.|[![](./images/button.png)](https://apps.axibase.com/chartlab/f95d6995)<br>[![](./images/button.png)](https://apps.axibase.com/chartlab/f95d6995/6/)|
```
Comment (rodionos) [inline widgets/text-widget/README.md]: When lines are long, a line break is better than comma to keep the setting an its value on one line.

**Comment by kghamilton89** [comment]
I'll close this PR and open two new ones: one with the changes you've pointed out (broken formatting) and another one with grammar changes.
